### DISCUSSION TITLE
Can't instance.setFilter with `false`

### DISCUSSION THREAD

**Discussion by laduke** [TanStack/table#2257] [General]
Hi, I might just be misunderstanding the API but

when I use the table instance 
`setFilter("authorized", false) `
or
`setAllFilters([{id: 'authorized', value: false ])`

both completely remove the 'authorized' filter from state.filters

but if I use the setFilter from the column Filter function, it does set the 'authorized' value to `false`.

**Comment by simplecommerce** [comment]
I have a similar issue, where I am not able to use `setFilters` from the column, to set my filter value to `false` or event `0`. It acts similar to when setting it to `undefined`.

I am not sure if this is normal behavior?

**Reply by simplecommerce** [reply]
Nevermind, it is because of this: https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js

In my column, even if I created a custom Filter, I for example for a boolean type, I had to use `filter: "equals"` instead, by default it is set to `text` and `text` removes `0` and `false`.
